### DISCUSSION TITLE
Bridge Mode

### DISCUSSION THREAD

**Discussion by g19i85g** [Freetz-NG/freetz-ng#480] [Allgemein]
Hallo,

ich möchte mein Fritzbox 7490 als reines Modem Betreiben ist das mit Freetz-NG möglich?

**Comment by fda77** [comment]
Auch ohne

**Reply by g19i85g** [reply]
Echt, wie denn? 
Nach allem was ich gelesen habe ist das nicht möglich weil die FB sich immer mit den Internet verbinden möchte und wenn man z. B. falsche zugangsdaten an gibt funktioniert es, aber die Verbindung ist schlecht oder bricht regelmäßig ab.

**Reply by fda77** [reply]
Nur weil einer dem anderen nachplappert und es dann irgend wann viele sagen heisst nicht dass da was dran ist ^^
Ich benutze eine 7490 so. Die Verbindung ist schlecht, das liegt aber an der Telekom ;-)
Zugangsdate brauch die Fritzbox keine, die kommen zb auf den Openwrt Router

![image](https://user-images.githubusercontent.com/471850/157124132-c1077fca-3d7e-42af-8c02-879561d642bc.png)

Diese 60 mbit sind laut toller Telekom "bis zu 100" , manchmal auch nur die Hälfte

**Reply by g19i85g** [reply]
Hm interessant, hast recht habe ich noch nicht getestet. Wie hast du es gemacht einfach bei DSL Anbieter andere ausgewählt, falsche Zugangsdaten eingegeben und gesagt das andere Geräte eine Internetverbindung aufbauen dürfen?

**Reply by fda77** [reply]
Es gibt die Option "keine Zugangsdaten". Diese sind nur für Pppoe erforderlich, das mach aber ein anderes Gerät

**Reply by g19i85g** [reply]
Ah cool danke das werde ich testen.

**Reply by fda77** [reply]
Du kannst die Fritzbox dann nur mit der internen IP ansprechen, oft 192.168.178.1, eine öffentlich hat sie nicht. Damit kann dise auch nichts verwenden was Internet benötigt, zb Myfritz, Updatecheck usw.
Wenn du einen guten Router und Ahnung hast, kannst du eine interne route anlegen die über den Router geht. Das hab ich für die ISDN Geräte so gemacht zu einem Asterisk

**Reply by g19i85g** [reply]
OK, das sollte kein Problem sein. Ich habe mir ein Mikrotik RB4011IGS geholt habe zwar noch nicht so viel Ahnung mit Netzwerk Technik werde es aber schon irgendwie hinbekommen.
